packages/studio/src/store/slices/createLayoutSlice.ts

```diff
@@ -4,7 +4,10 @@ import { SliceCreator } from "../models/utils";
 import removeTopLevelFragments from "../../utils/removeTopLevelFragments";
 
 const createLayoutSlice: SliceCreator<LayoutSlice> = () => ({
-  layouts: removeTopLevelFragments(initialStudioData.layoutNameToLayoutState),
+  // TODO (SLAP-2930): Remove top-level fragments from layouts
```

Comment:
Do we need this TODO any more? Seems like we are removing the top-level fragments from layouts.